slack/webhook/client.py

```diff
@@ -16,13 +17,21 @@ class WebhookClient:
     logger = logging.getLogger(__name__)
 
     def __init__(
-        self, url: str, default_headers: Dict[str, str] = {},
+        self,
+        url: str,
+        timeout: int = 30,
+        proxy: Optional[str] = None,
```

Comment:
I know this change is supposed to be done by #715 but allow me to include in this PR to avoid conflict.